tabs/configuration.js

```diff
@@ -456,6 +456,10 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             serialRXtypes.push('Spektrum Bidir SRXL');
         }
 
+        if (semver.gte(CONFIG.flightControllerVersion, "3.2.0"))  {
```

Comment:
@nathantsoi: I do not agree with you that your idea of 'transparency' is a good engineering practice if it means using a firmware release version for version gating of an API that is shared between multiple firmware projects.